spec/models/ability_spec.rb

```diff
@@ -192,29 +192,39 @@
       end
     end
 
-    context 'reader, editor, owner' do
+    context 'reader' do
       it 'not be allowed: to write' do
-        [reader, editor, owner].each do |role|
-          Ability.new(role, nil).should_not be_able_to(:write, item)
+        expect(Ability.new(reader, nil)).to_not be_able_to(:write, item)
+      end
+
+      it 'be allowed: to read' do
+        expect(Ability.new(reader, nil)).to be_able_to(:show, item)
+      end
+    end
+
+    context 'editor, owner' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.